libsigner/src/events.rs

```diff
@@ -211,6 +213,19 @@ impl EventReceiver for SignerEventReceiver {
                 return Err(EventError::Terminated);
             }
 
+            if request.method() == &HttpMethod::Get {
```

Comment:
not sure we have to check if it is a Get method. we could just put this request.url() check below like how "/stackerdb_chunks" and "/block_proposal" are done.

Reply:
Yep good call